site/_includes/partials/devtools/es/whats-new.md

```diff
@@ -5,16 +5,16 @@ Por favor, revisa la versión en inglés de <a href="/tags/new-in-devtools/" tra
 
 ### Chrome 105 {: #chrome105 }
 
-* [Step-by-step replay in the Recorder](/es/blog/new-in-devtools-105/#recorder)
-* [Support mouse over event in the Recorder panel](/es/blog/new-in-devtools-105/#recorder-hover)
-* [Largest Contentful Paint (LCP) in the Performance insights panel](/es/blog/new-in-devtools-105/#lcp)
-* [Identify flashes of text (FOIT, FOUT) as potential root causes for layout shifts](/es/blog/new-in-devtools-105/#foit-fout)
-* [Top layer badge in the Elements panel](/es/blog/new-in-devtools-105/#top-layer)
-* [Attach Wasm debugging information at runtime](/es/blog/new-in-devtools-105/#wasm)
-* [Support live edit during debugging](/es/blog/new-in-devtools-105/#live-edit)
-* [View and edit @scope at rules in the Styles pane](/es/blog/new-in-devtools-105/#scope)
-* [Sourcemap improvements](/es/blog/new-in-devtools-105/#sourcemaps)
-* [Miscellaneous highlights](/es/blog/new-in-devtools-105/#misc)
+* [Reproducción paso a paso en la grabadora](/es/blog/new-in-devtools-105/#recorder)
+* [Soporte de eventos de ratón en el panel de la grabadora](/es/blog/new-in-devtools-105/#recorder-hover)
+* [Largest Contentful Paint (LCP) en el panel de percepción de Rendimiento](/es/blog/new-in-devtools-105/#lcp)
```

Comment:
I'd suggest that we refer to strings that aren't translated in DevTools as they are displayed.

```suggestion
* [Largest Contentful Paint (LCP) en el panel Performance Insights](/es/blog/new-in-devtools-105/#lcp)

```